dart/lib/src/sentry.dart

```diff
@@ -108,9 +100,23 @@ class Sentry {
     _hub = Hub(options);
     hub.close();
 
-    // execute integrations after hub being enabled
-    for (final integration in options.integrations) {
-      await integration(HubAdapter(), options);
+    if (appRunner != null) {
```

Comment:
I prefer to keep the comments, they are useful when debugging/reading code.

> // execute integrations after hub being enabled

this one for example, was a bug on the java sdk